test/perf/ok/dao.drun-run.ok

```diff
@@ -0,0 +1,16 @@
+ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
+ingress Completed: Reply: 0x4449444c0000
+Ok: Reply: 0x4449444c086d016c08dbb7017db793e6d10307abfec2a4060591ecada00804d6a9bbae0a7cb4bfd4960b68f18d9dc10b078effd6e90e026c03e1edeb4a71b3c4b1f20468c7ebc4d009036d7b6b06ca87d0cd047ffefe8fdc0a7fb2d5bc880b7fe7fadec00c7fbd8bfbbf0e71e1aca5d90f7f6e066c02006801056c01b9ef9380087d010000
+Ok: Reply: 0x4449444c026c03e1edeb4a71b3c4b1f20468c7ebc4d009016d7b01000564756d6d79010a0000000000000000010100
+ingress Completed: Reply: 0x4449444c016b029cc2017de58eb402710100012c43616c6c6572206e6565647320616e206163636f756e7420746f207375626d697420612070726f706f73616c
+Ok: Reply: 0x4449444c016c01b9ef9380087d010000
+Ok: Reply: 0x4449444c026c03e1edeb4a71b3c4b1f20468c7ebc4d009016d7b0100087472616e73666572010a000000000000000001012a4449444c026c02fbca0168d8a38ca80d016c01b9ef9380087d0100010a00000000000000000101b0ea01
+ingress Completed: Reply: 0x4449444c016b029cc2017de58eb402710100012c43616c6c6572206e6565647320616e206163636f756e7420746f207375626d697420612070726f706f73616c
+Ok: Reply: 0x4449444c086d016c08dbb7017db793e6d10307abfec2a4060591ecada00804d6a9bbae0a7cb4bfd4960b68f18d9dc10b078effd6e90e026c03e1edeb4a71b3c4b1f20468c7ebc4d009036d7b6b06ca87d0cd047ffefe8fdc0a7fb2d5bc880b7fe7fadec00c7fbd8bfbbf0e71e1aca5d90f7f6e066c02006801056c01b9ef9380087d010000
+Ok: Reply: 0x4449444c036d016c02b3b0dac30368daae9c8f09026c01b9ef9380087d010000
+Ok: Reply: 0x4449444c086e016c08dbb7017db793e6d10307abfec2a4060591ecada00804d6a9bbae0a7cb4bfd4960b68f18d9dc10b078effd6e90e026c03e1edeb4a71b3c4b1f20468c7ebc4d009036d7b6b06ca87d0cd047ffefe8fdc0a7fb2d5bc880b7fe7fadec00c7fbd8bfbbf0e71e1aca5d90f7f6e066c02006801056c01b9ef9380087d010000
+Ok: Reply: 0x4449444c086e016c08dbb7017db793e6d10307abfec2a4060591ecada00804d6a9bbae0a7cb4bfd4960b68f18d9dc10b078effd6e90e026c03e1edeb4a71b3c4b1f20468c7ebc4d009036d7b6b06ca87d0cd047ffefe8fdc0a7fb2d5bc880b7fe7fadec00c7fbd8bfbbf0e71e1aca5d90f7f6e066c02006801056c01b9ef9380087d010000
```

Comment:
this is the response to `get_proposal 1`, but seems to be identical to above. @chenyan-dfinity this is suspicious

Reply:
Aaah, those are big-typed `null`s. I am to fix this.